src/test/java/org/opentestsystem/ap/imrt/common/builder/BaseItemBuilder.java

```diff
@@ -71,6 +71,9 @@
     private String testCategory = "interim";
     private String scoringEngine = "handScored";
     private String allowCalculator = "Yes";
+    private Boolean aslUploadedPriorToLastContentUpdate = false;
+    private Boolean brailleUploadedPriorToLastContentUpdate = false;
+    private Boolean closedCaptioningUploadedPriorToLastContentUpdate = false;
```

Comment:
Can you set one of these to "true" since false is the default value set in the db?